File: src/main/java/com/checkers/controllers/FinishController.java
package com.checkers.controllers;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("finish")
public class FinishController {

    @PostMapping
    public String getFinishPage() {
        return "finish";
    }
}

File: src/main/kotlin/com/checkers/controller/FinishController.kt
package com.checkers.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("finish")
class FinishController {

    @PostMapping
    fun getFinishPage(): String =
        "finish"
}
